src/System.Net.Mail/tests/Functional/SmtpClientTest.cs

```diff
@@ -36,7 +36,7 @@ private string TempFolder
             {
                 if (_tempFolder == null)
                 {
-                    _tempFolder = Path.Combine(Path.GetTempPath(), GetType().FullName, Guid.NewGuid().ToString());
+                    _tempFolder = Path.Combine(Path.GetTempPath(), GetType().Name + "_" + Path.GetRandomFileName());
                     if (Directory.Exists(_tempFolder))
```

Comment:
Which is GetRandomFileName better than Guid?

Reply:
Also, as long as we're changing these, can we just use FileTestCleanupBase as we do in other test suites?

Reply:
@danmosemsft mentioned in this https://github.com/dotnet/corefx/issues/28559#issuecomment-377128638 that it might be risky to make that change at this moment.

Reply:
I have changed it for System.Net.Mail (using FileCleanupTestBase). For the other files @danmosemsft mentioned it may to risky to change them at this point https://github.com/dotnet/corefx/issues/28559#issuecomment-377128638